### ISSUE TITLE
A11Y - Missing Caption or Summary for Color Coding Data Table

### ISSUE THREAD

**Issue by Keisha19881** [GSA/search-gov-website#346]
**URL:** https://search.gov/admin-center/dashboard/users.html

**BUG:** The data table that uses color (no color, yellow, red) to indicate user status ("Approved," "Pending email verification," "Pending approval," "Not approved"), lacks a descriptive caption or summary. NVDA is not able to provide an overview of the table's purpose and the meaning of the color coding.

**Fix:** Add a <caption> element as the first child of the <table> element. This caption should clearly explain the purpose of the table and the meaning of the color coding used.

![Image](https://github.com/user-attachments/assets/d5c25549-7fcd-4b03-8618-63f8ecd97f01)
